bin/clone-app

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env ruby
+
+class App
+  def initialize(name)
+    @name = name
+  end
+
+  def prepare!
+    Dir.chdir("apps") do
+      if File.directory?(name)
+        update
+      else
+        clone
+      end
+    end
+  end
+
+private
+
+  attr_reader :name
+
+  def clone
+    system "git clone -b #{branch} #{git_url}"
+  end
+
+  def has_local_changes?
+    !system("git diff --quiet --ignore-submodules --no-ext-diff")
+  end
+
+  def checkout
+    system "git checkout #{branch}"
+  end
+
+  def fetch_origin
+    system "git fetch origin"
+  end
+
+  def merge
+    system "git merge --ff-only origin/#{branch}"
+  end
+
+  def update
+    Dir.chdir(name) do
+      if has_local_changes?
+        puts "skipped updating #{name} due to local changes"
+        return true
```

Comment:
since this has a return true we can just have the if block ended rather than an else condition